ios/Podfile.lock

```diff
@@ -117,7 +117,7 @@ EXTERNAL SOURCES:
 
 SPEC CHECKSUMS:
   device_info_plus: c6fb39579d0f423935b0c9ce7ee2f44b71b9fce6
-  Flutter: f04841e97a9d0b0a8025694d0796dd46242b2854
```

Comment:
@diehuxx can you check which version of Flutter you're currently using? In this project Flutter is bundled with Hermit, so we should make sure it's using your Hermit version of Flutter vs. your machine version.